### PR TITLE
added soft deleted(endpoint)/JpaRepository/liquibase (create table bo…

### PR DESCRIPTION
**Pull request by OlexVoit**
…oks)

### CODE REVIEW

File: src/main/java/mate/academy/bookshop/model/Book.java [hunk 1]
```diff
@@ -8,23 +8,37 @@
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```
Comment (vadymhrnk) [inline src/main/java/mate/academy/bookshop/model/Book.java]: `@Where` is marked as deprecated, so instead you can use `@SQLRestriction("is_deleted=false")`

**Review by vadymhrnk — CHANGES_REQUESTED** [review]
Overall good job, but I believe you missed one endpoint for PUT method

> Book Endpoints:
> 
> - GET: /api/books (Retrieve book catalog) (should be done in the previous PR)
> - GET: /api/books/{id} (Retrieve book details) (should be done in the previous PR)
> - POST: /api/books (Create a new book) (should be done in the previous PR)
> - PUT: /api/books/{id} (Update a specific book)
> - DELETE: /api/books/{id} (Delete a specific book)

File: src/main/java/mate/academy/bookshop/model/Book.java (hunk 1 above)
Comment (OlexVoit) [reply src/main/java/mate/academy/bookshop/model/Book.java]: my bad

**Review by kshuryhin — APPROVED** [review]
Looks good!

**Pull request merged**